rest-api/src/main/resources/definitions/user_consent_history.yml

```diff
@@ -19,7 +19,6 @@ properties:
         description: ISO 8601 date string (e.g. "YYYY-MM-DD").
     imageData:
         type: string
-        format: byte
```

Comment:
Um, I thought you wanted to enforce the correct base64 encoding, which the byte[] format would do...

Reply:
No, it's the opposite. Right now, the client assumes it's a Base64 encoded string. But it isn't. So the client just crashes when it tries to decode it and fails.